Preferences.tsx

```diff
@@ -36,14 +41,20 @@ export const PreferencesProvider: React.FC<PreferencesProviderProps> = ({childre
   const [preferences, setFullPreferences] = useState<Preferences>(defaultPreferences);
 
   useAsyncEffect(async () => {
-    const storedPreferences = await AsyncStorage.getItem('preferences');
+    let storedPreferences = {};
+    try {
+      storedPreferences = preferencesSchema.parse(JSON.parse((await AsyncStorage.getItem(PREFERENCES_KEY)) ?? '{}'));
+    } catch (_e) {
```

Comment:
Will Sentry pick up this exception and send it to us? If not, can we log it? I imagine we will want to dig into those